src/main/java/org/candlepin/model/OwnerInfoBuilder.java

```diff
@@ -32,6 +33,14 @@ public class OwnerInfoBuilder {
 
     public OwnerInfoBuilder() {
         ownerInfo = new OwnerInfo();
+        ownerInfo.setConsumerTypeCountByPool(new LinkedHashMap<>());
```

Comment:
same question here

Reply:
Looking back at candlepin-4.0-HOTFIX, empty maps has been the correct behaviour here, so all good